app/mailers/hackathons/submission_mailer.rb

```diff
@@ -0,0 +1,22 @@
+class Hackathons::SubmissionMailer < ApplicationMailer
+  before_action :set_hackathon
+
+  def confirmation
+    mail to: @hackathon.applicant.email_address, subject: "Your hackathon was submitted!"
+  end
+
+  def notification
```

Comment:
should we call this `admin_notification`? it's a little more clear what the notification is for